controlsfx/src/main/java/impl/org/controlsfx/skin/AutoCompletePopupSkin.java

```diff
@@ -42,7 +45,25 @@
     private final ListView<T> suggestionList;
     final int LIST_CELL_HEIGHT = 24;
 
-    public AutoCompletePopupSkin(AutoCompletePopup<T> control){
+    public AutoCompletePopupSkin(AutoCompletePopup<T> control) {
+        this(control, control.getConverter());
+    }
+
+    /**
+     * @param displayConverter
+     *            An alternate {@link StringConverter} to use. This way, you can show autocomplete suggestions that when applied will fill in a
+     *            different text than displayed. For example, you may preview {@code Files.newBufferedReader(Path: path) - Bufferedreader} but
+     *            only fill in {@code Files.newBufferedReader(}
+     */
```

Comment:
The Javadoc is not well formated. You should add the description first, then add every param (there are some missing) and a description for every one if possible.

Also, try to have the line not too long, you can use the auto-format on NetBeans or IntellijIdea on this portion of code to have it well-formatted.